js-jasmine/src/agedBrie.js

```diff
@@ -0,0 +1,15 @@
+class AgedBrie {
```

Comment:
good idea to put Aged Brie and Backstage passes in their own class to split up the logic. Overall, the solution could do with some more refactoring, but this is why you have put it as incomplete!